### DISCUSSION TITLE
'In Range' Indicator

### DISCUSSION THREAD

**Discussion by heymish96** [runelite/runelite#16735] [Ideas]
Been learning the inferno recently and something im always strugling with is knowing if a target is in range or not. 

Would it be possible to make a plugin that checks what weapon you're using and style then when you hover over an npc it could, for example, highlight the npc green or red.

Situation example, im north of north pillar, mager is to the west and everything else is stacked to the south. If i attack the mager and get pulled out from behind the pillar I'm in the weeds, but if i can chill and hit it with a bow on longrange from there I'm chillin.

I've used the line pof sight plugin and it definitely helps a bit but need to manually set it to each distance. Cheers!

**Comment by pwatts6060** [comment]
Line Of Sight hub plugin can help with this. Doesn't do exactly what you're saying but you could set it to like 9 tiles and can count tiles from there.

**Comment by Nightfirecat** [comment]
There's a "Radius Markers" plugin that could achieve this on the plugin hub
